### PR TITLE
Support babel transpiling when generating html and updating snippets

### PR DESCRIPTION
**Pull request by sunesimonsen**
This PR fixes the final part of #2, it uses babel for transpiling if babel core can be required.

### CODE REVIEW

**Comment by sunesimonsen** [comment]
mhh this might not work so well if there is no .babelrc file. I'll just test it.

File: lib/evaluateSnippets.js [hunk 1]
```diff
@@ -3,6 +3,24 @@ var async = require('async');
...
+    var babelOptions = readBabelrc();
+
+    transpile = function transpile(code) {
+        var babelResult = babelCore.transform(code, extend({}, babelOptions, {
+            sourceMaps: false,
+        }));
+        return babelResult.code;
+    };
+} catch (e) {
+  transpile = function transpile (code) {
```
Comment (papandreou) [inline lib/evaluateSnippets.js]: indent?

**Comment by papandreou** [comment]
:+1:

We should add some quick tests.

**Comment by sunesimonsen** [comment]
it is getting sad, I'm fixing it.

File: lib/evaluateSnippets.js (hunk 1 above)
Comment (sunesimonsen) [inline lib/evaluateSnippets.js]: Sorry, I'll go through an see if I left out some semicolons as well :-)

**Pull request merged**